azure-go-aci/main.go

```diff
@@ -48,12 +48,7 @@ func main() {
 			return err
 		}
 
-		ctx.Export("containerIPv4Address", containerGroup.IpAddress.ApplyT(func(ip *containerinstance.IpAddressResponse) (string, error) {
-			if ip == nil || ip.Ip == nil {
-				return "", nil
-			}
-			return *ip.Ip, nil
-		}).(pulumi.StringOutput))
+		ctx.Export("containerIPv4Address", containerGroup.IpAddress.Ip())
```

Comment:
The ApplyT used to coerce `*string` to `string` by interpreting nil as "". IS that still happening? Does `ctx.Export` work with nils? Have you had a chance to test this? The CI does not test all  examples.

Reply:
```
kdixler@kdixler-20xw004aus ~/GolandProjects/examples/azure-go-aci (git)-[dixler/1115/refactor-azure-go-binless] % pulumi up
Previewing update (issue1115)

View Live: https://app.pulumi.com/kyle/azure-go-aci/issue1115/previews/c49fb6bf-3e9c-4581-b200-56878bd789e1

     Type                                              Name                    Plan       
 +   pulumi:pulumi:Stack                               azure-go-aci-issue1115  create     
 +   ├─ azure-native:resources:ResourceGroup           aci-rg                  create     
 +   └─ azure-native:containerinstance:ContainerGroup  helloworld              create     
 
Resources:
    + 3 to create

Do you want to perform this update? yes
Updating (issue1115)

View Live: https://app.pulumi.com/kyle/azure-go-aci/issue1115/updates/6

     Type                                              Name                    Status      
 +   pulumi:pulumi:Stack                               azure-go-aci-issue1115  created     
 +   ├─ azure-native:resources:ResourceGroup           aci-rg                  created     
 +   └─ azure-native:containerinstance:ContainerGroup  helloworld              created     
 
Outputs:
    containerIPv4Address: "20.120.145.148"

Resources:
    + 3 created

Duration: 1m11s
```